recipes/discasm/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.2" %}
+
+package:
+    name: discasm
+    version: {{ version }}
+
+source:
+  fn: DISCASM-v{{ version }}.tar.gz # [unix]
```

Comment:
You can drop the `[unix]` selectors: those are always true since Bioconda only builds packages for Linux and MacOS both of them being "unix".

Reply:
done